src/cloudformation_cli_python_lib/resource.py

```diff
@@ -232,7 +253,20 @@ def __call__(
                 )
             invoke = True
             while invoke:
-                progress = self._invoke_handler(caller_sess, request, action, callback)
+                metrics.publish_invocation_metric(datetime.now(), action)
```

Comment:
for all dates, do we expect them to be local time or UTC? Java's `Instant.now()` also returns UTC AFAIK.

in either case, does `datetime.now()` and `datetime.utcnow()` attach the correct TZ information for CloudWatch to interpret? i don't think so, but i also don't think we necessarily need pytz from python 3.2 onwards thanks to [datetime.timezone](https://docs.python.org/3/library/datetime.html#datetime.timezone)

```
>>> from datetime import datetime, timezone
>>> print(datetime.now(), "#", datetime.utcnow(), "#", datetime.now(timezone.utc))
2019-12-02 11:16:39.168110 # 2019-12-02 19:16:39.168116 # 2019-12-02 19:16:39.168117+00:00
```

for the invoke time, it's still fine to use the timezone-less variant.

Reply:
did some testing, `datetime.utcnow()` seems to work fine

Reply:
switched them all to utcnow for consistency.